Mage.Sets/src/mage/cards/o/OmnathLocusOfAll.java

```diff
@@ -0,0 +1,140 @@
+import mage.abilities.effects.OneShotEffect;
+import mage.abilities.effects.mana.AddManaInAnyCombinationEffect;
+import mage.cards.Card;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.cards.CardsImpl;
+import mage.constants.*;
+import mage.game.Game;
+import mage.players.Player;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+public class OmnathLocusOfAll extends CardImpl {
+    public OmnathLocusOfAll(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.CREATURE}, "{W}{U}{B/P}{R}{G}");
+        this.addSuperType(SuperType.LEGENDARY);
+        this.addSubType(SubType.PHYREXIAN);
+        this.addSubType(SubType.ELEMENTAL);
+        this.power = new MageInt(4);
+        this.toughness = new MageInt(4);
+
+        //If you would lose unspent mana, that mana becomes black instead.
+        this.addAbility(new SimpleStaticAbility(new OmnathLocusOfAllManaEffect()));
+
+        //At the beginning of your precombat main phase, look at the top card of your library. You may reveal that card
+        //if it has three or more colored mana symbols in its mana cost. If you do, add three mana in any combination of
+        //colors and put it into your hand. If you don’t reveal it, put it into your hand.
+        this.addAbility(new BeginningOfPreCombatMainTriggeredAbility(
+                new OmnathLocusOfAllCardEffect(), TargetController.YOU, false
+        ));
+    }
+
+    private OmnathLocusOfAll(final OmnathLocusOfAll card) {
+        super(card);
+    }
+
+    @Override
+    public OmnathLocusOfAll copy() {
+        return new OmnathLocusOfAll(this);
+    }
+}
+
+class OmnathLocusOfAllManaEffect extends ContinuousEffectImpl {
+
+    OmnathLocusOfAllManaEffect() {
+        super(Duration.WhileOnBattlefield, Layer.RulesEffects, SubLayer.NA, Outcome.Benefit);
+        staticText = "if you would lose unspent mana, that mana becomes black instead";
+    }
+
+    private OmnathLocusOfAllManaEffect(final OmnathLocusOfAllManaEffect effect) {
+        super(effect);
+    }
+
+    @Override
+    public OmnathLocusOfAllManaEffect copy() {
+        return new OmnathLocusOfAllManaEffect(this);
+    }
+
+    @Override
+    public boolean apply(Game game, Ability source) {
+        Player player = game.getPlayer(source.getControllerId());
+        if (player != null) {
+            player.getManaPool().setManaBecomesBlack(true);
+        }
+        return true;
+    }
+}
+
+class OmnathLocusOfAllCardEffect extends OneShotEffect {
+
+    public OmnathLocusOfAllCardEffect() {
+        super(Outcome.Benefit);
+        this.staticText = "look at the top card of your library. You may reveal that card if it has three or more " +
+                "colored mana symbols in its mana cost. If you do, add three mana in any combination of colors and " +
+                "put it into your hand. If you don’t reveal it, put it into your hand.";
```

Comment:
The apostrophe in "don’t" here is what broke the Verify Cards test, switching it to a normal apostrophe character like ' will fix it.

Reply:
oh yeah... not the first time... comes from copy pasting the rule text from scryfall